coordinator/README.md

```diff
@@ -11,11 +11,27 @@ npm install
 npm run serve
 ```
 
+### Directories
+- `/models` Typescript interface/class definitions
+- `/resources` mustache template files for xml responses
+- `/routes` API endpoint definitions
+- `/services` FHIR translations
+  - `/formatters` builds xml
+  - `/handlers` defines spine and sandbox handlers - how each of the endpoints respond to requests
+  - `/serialisation` xml serialisation
+  - `/translation` conversion of valid HL7 FHIR message to HL7V3 ParentPrescription message
+  - `/validation` incoming FHIR payload validation on API request
+
 ## Deployment
 
 Redeploy the API Proxy. See the main [README.md](../README.md).
 
 ## Endpoints
 
-- [ ] POST `/ConvertFullMessage`
-- [ ] POST `/ConvertSignatureFragments`
+Endpoints are found in `/src/routes`.
```

Comment:
You've not mentioned the `/src` directory in the file paths in the Directories section but you have here. Maybe just write `/routes` for consistency with above?

Reply:
was thinking about putting `/src` in the directories section tbh, maybe that would make this better?

Reply:
yh that looks good